core/devdoc/gateway_createfromjson_requirements.md

```diff
@@ -118,3 +118,30 @@ Gateway_CreateFromJson creates a new gateway using information contained within
 **SRS_GATEWAY_JSON_17_002: [** This function shall return `NULL` if starting the gateway fails. **]**
 
 **SRS_GATEWAY_JSON_14_008: [** This function shall return `NULL` upon any memory allocation failure. **]**
+
+
+## Gateway_UpdateFromJson
+```
+extern int Gateway_UpdateFromJson(GATEWAY_HANDLE gw, const char* json_content);
+```
+Gateway_UpdateFromJson adds loades, modules or links based on a well-formed JSON configuration content.
+
+**SRS_GATEWAY_JSON_04_003: [** If `json_content` is NULL the function shall return NULL. **]**
```

Comment:
Fixed.

---
In reply to: [108959788](https://github.com/Azure/azure-iot-gateway-sdk/pull/189#discussion_r108959788) [](ancestors = 108959788)